pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -390,6 +410,20 @@ func (r *RestartReconciler) restartCluster(ctx context.Context, downPods []*PodF
 	opts := []startdb.Option{
 		startdb.WithInitiator(r.InitiatorPod, r.InitiatorPodIP),
 	}
+	// If a communal path is set, include all of the EON parameters.
+	if r.Vdb.Spec.Communal.Path != "" {
+		// build communal storage params if there is not one
+		if r.ConfigurationParams == nil {
+			res, err := r.ConstructConfigParms(ctx)
+			if err != nil {
```

Comment:
Same comment here. Let's use the helper to check `res`
```suggestion
			if verrors.IsReconcileAborted(res, err) {
```

Reply:
After applying this change , the e2e test(https://github.com/vertica/vertica-kubernetes/actions/runs/6355160688/job/17267471737) "offline-upgrade-ks-0" keep failing on 75-wait-for-upgrade which said OfflineUpgradeInProgress should be true. Some res(when err == nil) might abort the restart reconciler after the offline upgrade. Need Matt's help to fix this.